docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_YOLACT.md

```diff
@@ -136,7 +136,9 @@ git clone https://github.com/dbolya/yolact
 git checkout 57b8f2d95e62e2e649b382f516ab41f949b57239
 ```
 
-**Step 2**. Download a pre-trained model, for example `yolact_base_54_800000.pth`.
+3. Setup environment as described in `README.md`.
+
+**Step 2**. Download a pre-trained model from list attached in section `Evaluation` from `README.md` document, for example `yolact_base_54_800000.pth`.
```

Comment:
from list attached in section `Evaluation` from `README.md` document ---> from the list attached in the `Evaluation` section of `README.md` document